app/src/store/games.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+import Api from "../api";
+
+export const GET_GAMES = "GET_GAMES";
+
+export const getGames = dispatch =>
+    Api.games().then((response) => {
+        if (response.ok) {
+            response.json().then((gamesList) => {
+                dispatch({
+                    type: GET_GAMES,
+                    payload: {
+                        gamesList
+                    }
+                });
+            });
+        }
+    });
+
+// ------------------------------------
+// Reducer
+// ------------------------------------
+/* eslint-disable no-param-reassign */
+export default function gamesReducer(state = null, { type, payload }) {
+    if (type === GET_GAMES) {
+        state = {
```

Comment:
if there aren't any other objects that you would like to keep in store then you can replace `state = { gamesList: ... }` with `state = gamesList` to keep it simple